frontend/Panel.js

```diff
@@ -234,7 +239,16 @@ class Panel extends React.Component {
       );
     }
     if (!this.state.isReact) {
-      return <div style={styles.loading}><h2>Looking for React…</h2></div>;
+      return (
+        <div style={styles.loading}>
+          <h2>Looking for React…</h2>
+          {showFileUrlWarningMessage &&
+            <div style={styles.troubleshootingHint}>
+              Please check if the devtools extension has 'Allow access to file URLs' permission
```

Comment:
Can we change this to [this wording](https://github.com/facebook/react-devtools/blob/60c3e1bfa3ce60a22f1de1f3d8ea2beda3df2a2e/shells/chrome/popups/not-detected.html#L6) and somehow open that link in a new tab if it's clicked?